archlinux/PKGBUILD

```diff
@@ -1,26 +1,56 @@
 # Maintainer : Christian Rebischke <chris.rebischke@archlinux.org>
 
 pkgname=caddy
-pkgver=2.0.0beta10
-_pkgver=2.0.0-beta10
+pkgver=2.0.0beta13
```

Comment:
Dunno if it matters, but we had to make the tag `v2.0.0beta.13` in order to make Go modules happy...

Reply:
Cool. Then I can just use `pkgver` in the PKGBUILD and don't need to track this twice. Thanks a lot.

Reply:
Ok, sorry. Looks like I misunderstood you here :'D. It doesn't matter for us here, because I just pull your code by commit (for checking your signature on this commit). I am not sure if pacman (the Arch Linux package manager can handle versions like `v2.0.0-beta.13`... we can try, but I can't promise. Or is it okay to leave it as `v2.0.0-beta13`?

Reply:
That's okay. I wasn't sure if the version had to be the same as in the Caddy repo.